pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -256,7 +280,7 @@ func (m *kubeGenericRuntimeManager) GetPods(all bool) ([]*kubecontainer.Pod, err
 
 		converted, err := m.toKubeContainer(c)
 		if err != nil {
-			glog.Warningf("Convert %s container %v of pod %q failed: %v", m.runtimeName, c, labelledInfo.PodUID, err)
+			glog.V(4).Infof("Convert %s container %v of pod %q failed: %v", m.runtimeName, c, labelledInfo.PodUID, err)
```

Comment:
Because kubelet lists containers frequently and as long as there is one corrupt container, kubelet will log over and over again. Since we eventually ignore the container, we don't want it to cause too much spam.